pkg/buildkit/buildkit.go

```diff
@@ -38,6 +40,11 @@ func (b *clientBuilder) WithDockerAuthConfig(configDir string) *clientBuilder {
 	return b
 }
 
+func (b *clientBuilder) WithSecrets(secrets map[string][]byte) *clientBuilder {
+	b.secrets = secrets
+	return b
+}
+
```

Comment:
I know we only use a newly-constructed client "once" but do you think this should this be a builder option or a part of the `BuildOptions` passed to `Build` since mounting is build-specific. I only ask because I was thinking about moving the `WithDockerAuthConfig` out of the builder and into the build options since that is less client-specific.

Reply:
I forgot we should be loading the files per-build (since configmaps can change), so I think it's totally reasonable for this to be on `Build`. Is 31dfb37 what you had in mind?

Reply:
> Is https://github.com/dominodatalab/hephaestus/commit/31dfb371e0233606f6a3dc59dcb1c15f21fb18a3 what you had in mind?

Yes sir.